set_fm_data.py

```diff
@@ -81,7 +81,20 @@ def initialize(self):
         # ToDo: AJO 2022-12-30: This code is copied in several modules: combine!
         self.CAR_MIN_SOC_IN_PERCENT = int(float(self.args["car_min_soc_in_percent"]))
         # Make sure this value is between 10 en 30
-        self.CAR_MIN_SOC_IN_PERCENT = max(min(30, self.CAR_MIN_SOC_IN_PERCENT), 10)
+        notification_message = ""
+        if self.CAR_MIN_SOC_IN_PERCENT < 10:
+            notification_message = f"Setting for minimum SoC (car_min_soc_in_percent) {self.CAR_MIN_SOC_IN_PERCENT} " \
+                                   f"in secrets.yaml too low. Using minimum value 10."
+            self.CAR_MIN_SOC_IN_PERCENT = 10
+        elif self.CAR_MIN_SOC_IN_PERCENT > 30:
+            notification_message = f"Setting for minimum SoC (car_min_soc_in_percent) {self.CAR_MIN_SOC_IN_PERCENT}" \
+                                   f" in secrets.yaml too high. Using maximum value 30."
+            self.CAR_MIN_SOC_IN_PERCENT = 30
+
+        if notification_message != "":
+            self.call_service('persistent_notification/create', message=notification_message,
+                              title="V2g Liberty configuration", notification_id="config_error")
```

Comment:
The `notification_id` could be more custom to the V2G-Liberty app, as a generic string might clash with persistent notifications from other HA apps. I suggest: `"v2g_liberty_config_error"`.

From [the docs](https://www.home-assistant.io/integrations/persistent_notification/):
> If notification_id is given, it will overwrite the notification if there already was a notification with that ID.

Reply:
Agree, will change